### PR TITLE
Andrew's Super Sweet Web Page

### PR DESCRIPTION
**Pull request by andrewborstein**
This page is da best evah.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,10 @@
 <body>
   <div class="center reamde">
     <h1> Spring 2015!</h1>
-    <a href="david.html">David Zabner's page</a>
+    <ul>
+    	<li><a href="david.html">David Zabner's page</a></li>
```
Comment (nw0428) [inline index.html]: This puts bullet points on the far right side of the page! 
Maybe there is a better way to list these...?

Also the index page is so lacking in style! Could you make it less ugly?

**Comment by nw0428** [comment]
Maybe use: http://developer.wordnik.com/docs.html or something similar to generate random words every time?

File: index.html (hunk 1 above)
Comment (andrewborstein) [inline index.html]: I noticed the bullets but figured they were keeping in line with the ugly, er, utilitarian theme that you already had going. Happy to make the index page prettier.

**Comment by andrewborstein** [comment]
Oh, I like that idea. And it would be a fun thing to try and implement. Either random words on each page load, or random words when interacted with. Like every time you click a word, it changes. Or maybe click a 'randomize' button that changes all the chosen words. I'm guessing that would have to be AJAX. Good idea, though! I'll look into it tonight.

**Comment by andrewborstein** [comment]
This pull request is ready for primetime, @nw0428. I added a MadLibs function to my profile section and made the main index page OOhhHhhHh pretty.

File: js/andrew/randomword.js [hunk 2]
```diff
@@ -0,0 +1,287 @@
+var wordList = [
```
Comment (nw0428) [inline js/andrew/randomword.js:1]: Slightly inelegant to just have the dictionary here but otherwise very nicely done!

**Comment by nw0428** [comment]
One comment and then LGTM
aka LGTM aka "Looks good to me"

File: js/andrew/randomword.js (hunk 2 above)
Comment (andrewborstein) [inline js/andrew/randomword.js:1]: Yes, it was a time saver. Found the word list online and just plopped it in.

**Comment by andrewborstein** [comment]
Also, did you see the pretty [index page](https://www.dropbox.com/s/se33mvx64r44zdf/Screenshot%202015-03-03%2017.15.35.png?dl=0)? Branding inspired by [Startup Institute](http://startupinstitute.com).

**Comment by MicahMartin** [comment]
Masterful Responsive Design. Millions of look goods to me

**Comment by MicahMartin** [comment]
lookgoodtome

**Comment by nickemmons** [comment]
jlawthumbsupgit

**Pull request merged**